### ISSUE TITLE
Try to setting up a VS Code debugger

### ISSUE THREAD

**Issue by vadim9999** [LLazyEmail/markdown-to-email#1385]
Maybe Arthur created his functions for debugging this app instead of using vs code debugger but now there is a problem: on the first look vs code debugger doesn't want work in our implementation - running build and then running code from dist folder

**Comment by atherdon** [comment]
yeah, for some reason - probably after upgrading some packages - we are running our build script not `node src/index.js`

now I see only 2 options - submodules and converting our rollup config into nextjs API that will call our generator.

**Comment by vadim9999** [comment]
submodules not solves the problem. 
the second option it will be in future but generator should be easy to debug and run in dev mode. I mean made changes -> rapidly compiles code again -> parses content again -> writes to the html file
